design/proxy.md

```diff
@@ -0,0 +1,90 @@
+# Proxies
+
+The installer should work behind a proxy.
+
+## Goals
+
+- Proxy installs complete successfully.
+- Docker can pull images from remote and local registries.
+
+## Non Goals
+
+- Automatically detect proxy from environment.
+- Validate kotsadm add-on works with a proxy.
+- Support docker versions below 19.03.
```

Comment:
outside of scope im sure, but has anyone asked for ipv6 support for anything kURL related?

Reply:
no, I haven't heard any requests yet